### PR TITLE
Feat/create filter for vets

### PR DESCRIPTION
**Pull request by sanderpa**

### CODE REVIEW

**Review by DimaMirk — APPROVED** [review]

File: client/src/app/vets/components/vets-filter/vets-filter.component.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		outline: none;
+	}
+	&:focus .form-filter__label {
+		transform: translate(10%, -110%);
+	}
+}
+.form-filter__label {
+	position: absolute;
+	transform: translate(20%, 10%);
+	color: #4788b6;
```
Comment (Samwa-ua) [inline client/src/app/vets/components/vets-filter/vets-filter.component.scss]: You should use variables

File: client/src/app/vets/components/vets-filter/vets-filter.component.scss [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	}
+}
+.form-filter-btn {
+	width: 150px;
+	height: 50px;
+	margin: 0 auto;
+
+	border: 2px solid #4788b6;
+	border-radius: 3px;
+	color: #4788b6;
```
Comment (Samwa-ua) [inline client/src/app/vets/components/vets-filter/vets-filter.component.scss]: Same here

File: client/src/app/vets/components/vets-filter/vets-filter.component.scss [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	background-color: inherit;
+	font-size: 20px;
+	&:hover {
+		color: #d68427;
```
Comment (Samwa-ua) [inline client/src/app/vets/components/vets-filter/vets-filter.component.scss]: Variables

File: client/src/app/vets/components/vets-filter/vets-filter.component.scss [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+		cursor: pointer;
+		background-color: #60b4f034;
+	}
+}
+
+// filter result
+
+.card-filter-title {
+	text-align: center;
+	color: #4788b6;
```
Comment (Samwa-ua) [inline client/src/app/vets/components/vets-filter/vets-filter.component.scss]: Variables

File: client/src/app/vets/components/vets-filter/vets-filter.component.scss [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+	margin: 0 auto;
+
+	border: 2px solid #4788b6;
+	border-radius: 3px;
+	color: #4788b6;
+	background-color: inherit;
+	font-size: 20px;
+	&:hover {
+		color: #d68427;
+		border-color: #d68427;
```
Comment (Samwa-ua) [inline client/src/app/vets/components/vets-filter/vets-filter.component.scss]: Variables

**Review by Samwa-ua — APPROVED** [review]

File: client/src/app/vets/components/vets-filter/vets-filter.component.scss (hunk 5 above)
Comment (sanderpa) [reply client/src/app/vets/components/vets-filter/vets-filter.component.scss]: done

File: client/src/app/vets/components/vets-filter/vets-filter.component.scss (hunk 2 above)
Comment (sanderpa) [reply client/src/app/vets/components/vets-filter/vets-filter.component.scss]: done

File: client/src/app/vets/components/vets-filter/vets-filter.component.scss (hunk 4 above)
Comment (sanderpa) [reply client/src/app/vets/components/vets-filter/vets-filter.component.scss]: done

**Pull request merged**